vertical-pod-autoscaler/pkg/admission-controller/logic/server_test.go

```diff
@@ -0,0 +1,284 @@
+package logic
+
+import (
+	"encoding/json"
+	"fmt"
```

Comment:
nit: can you have the standard imports block first and then the k8s.io block?

Reply:
Done.